includes/Checker/Checks/Plugin_Readme_Check.php

```diff
@@ -0,0 +1,153 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Checker\Checks\Plugin_Readme_Check
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Checker\Checks;
+
+use WordPress\Plugin_Check\Checker\Check_Result;
+
+/**
+ * Check the plugins readme.txt file and contents.
+ *
+ * @since n.e.x.t
+ */
+class Plugin_Readme_Check extends Abstract_File_Check {
+
+	/**
+	 * Check the readme.txt file.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param Check_Result $result The Check Result to amend.
+	 * @param array        $files  Array of plugin files.
+	 */
+	protected function check_files( Check_Result $result, array $files ) {
+		// Find the readme file.
+		$readme = self::filter_files_by_regex( $files, '/readme\.txt$/' );
```

Comment:
This check is wrong. It is possible to just use readme.md. See https://meta.trac.wordpress.org/ticket/403 && https://johnblackbourn.com/readme-md-plugin-on-wordpress-org/

Reply:
@spacedmonkey, Could we consider adding support for `readme.md` as an enhancement to this feature as suggest by @swissspidy?

Reply:
I think we should. First check to see if there is a readme.txt ( as they are most common ), then check to see if readme.md, only then error.

Reply:
I would do that in a separate issue/PR

Reply:
Can one of you review/approve the PR so we can merge it. Thanks!

Reply:
Isn't this just a matter of changing the regex, to check for readme.txt and readme.md.

Reply:
Probably, but it also needs tests. I don't see an issue with iterating quickly and adding that check in a separate PR.

Reply:
@mukeshpanchal27 @jjgrainger If you create a ticket / PR for checking for readme.md, we can unblock this and approve this PR.

Reply:
@spacedmonkey Open followup issue and fix phpmd notices.